drivers/tmp006/include/tmp006.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2014 PHYTEC Messtechnik GmbH
+ *               2017 HAW Hamburg
```

Comment:
Why moving this file from `drivers/include` to `drivers/tmp006/include` ? I think drivers public API definition should remain in the former.